src/test/groovy/com/onesignal/androidsdk/MainTest.groovy

```diff
@@ -1251,10 +1316,17 @@ class MainTest extends Specification {
         GradleTestTemplate.buildArgumentSets[GRADLE_LATEST_VERSION] = [['build']] // , '--info']]
         when:
         def results = runGradleProject([
+            'android.useAndroidX': true,
+            'android.enableJetifier': true,
```

Comment:
If my other questions are a "No" then when does `'android.useAndroidX': true,` require `'android.enableJetifier': true,` as well?

Reply:
`useAndroidX` is required if any AndroidX library is included. `enableJetifier` is only required if there is a mix of AndroidX and Android Support Libraries used in the project by any pre-compiled dependency such as OneSignal.

Also sorry didn't mean to commit this, I was just testing. Fixed!